pkg/provider/types/parameters.go

```diff
@@ -42,6 +43,12 @@ func GetUserAssignedIdentityID(parameters map[string]string) string {
 
 // GetTenantID returns the tenant ID
 func GetTenantID(parameters map[string]string) string {
+	// ref: https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/857
+	tenantID := strings.TrimSpace(parameters["tenantID"])
+	if tenantID != "" {
+		return tenantID
+	}
+	klog.V(2).Info("tenantId is deprecated and will be removed in a future release. Use 'tenantID' instead")
```

Comment:
PR description says log level is set to 3. Should we update here?

Reply:
updated it to log level 3